expression/builtin_time_test.go

```diff
@@ -1159,3 +1159,26 @@ func (s *testEvaluatorSuite) TestGetFormat(c *C) {
 		c.Assert(result, Equals, test.expect)
 	}
 }
+
+func (s *testEvaluatorSuite) TestTimeToSec(c *C) {
+	tests := []struct {
+		t      string
+		expect int64
+	}{
+		// Test case from https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_time-to-sec
+		{"22:23:00", 80580},
+		{"00:39:38", 2378},
+		// Test case for boundary values
+		{"00:00:00", 0},
+		{"23:59:59", 86399},
```

Comment:
I add a `00:00` test case, and `-2:-2:-2` will get a `invalid time format` error throw by ParseDuration.